src/models/hub/Directory.yaml

```
@@ -0,0 +1,27 @@
+          description: Filter applied (e.g., genre=146)
+        tag:
+          type: string
+          description: Tag name (e.g., Action)
+        tagType:
+          type: integer
+          description: Type of tag
+        count:
+          type: integer
+          description: Number of items matching the tag
+description: |
+  An array of items or sections which aren't actual media items, but are related to the search query.  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)